corehq/apps/cloudcare/static/cloudcare/js/formplayer/router.js

```diff
@@ -277,10 +277,8 @@ FormplayerFrontend.on("persistentMenuSelect", function (selections) {
     });
 });
 
-export default {
-    start: function () {
-        return new Router({
-            controller: API,
-        });
-    },
-};
+export function start() {
```

Comment:
i'm guessing this was changed because 
```
export default {
    start: function () {
        return new Router({
            controller: API,
        });
    },
};
``` 
didn't work? I 'm seeing other files that are exported and called in ways similar to router. For example, `corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js` is called via
```
export default {
    getIx: getIx,
    getForIx: getForIx,
    Form: function (json) {
        return new Form(json);
    },
    Question: function (json, parent) {
        return new Question(json, parent);
    },
    removeSiblingsOfRepeatGroup: removeSiblingsOfRepeatGroup,
};
```

and called like this `formUI.getForIx(form, ix)`. Is this fine?